src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -43,11 +43,13 @@ public final class JavadocTokenTypes {
      * <pre>{@code @return true if file exists}</pre>
      * <b>Tree:</b>
      * <pre>{@code
-     *   |--JAVADOC_TAG[4x3] : [@return true if file exists]
-     *       |--RETURN_LITERAL[4x3] : [@return]
-     *       |--WS[4x10] : [ ]
-     *       |--DESCRIPTION[4x11] : [true if file exists]
-     *           |--TEXT[4x11] : [true if file exists]
+     * JAVADOC_TAG -&gt; JAVADOC_TAG
```

Comment:
`-&gt;` in html is weird, it should be `->`.
Please investigate why we needed html escaping in Tokentypes.java but don't need here.

Reply:
@romani ,
I believe this is because we use a paragraph code block inside javadoc in JavadocTokenTypes.java ( `@code {` )
Whereas, in TokenTypes.java , the paragraph tag is used plainly and any '>' or '<' or even '@' are treated as HTML tags

From TokenTypes.java : 
```
     * <p>For example:</p>
     * <pre>
     * public class MyClass {
     *     private int foo;
     *     {foo = 10;}
     * }
     * </pre>
```

From JavadocTokenTypes.java : 
```
     * <p><b>Example:</b></p>
     * <pre>{@code @return true if file exists}</pre>
     * <b>Tree:</b>
     * <pre>{@code
     *   |--JAVADOC_TAG[4x3] : [@return true if file exists]
     *       |--RETURN_LITERAL[4x3] : [@return]
     *       |--WS[4x10] : [ ]
     *       |--DESCRIPTION[4x11] : [true if file exists]
     *           |--TEXT[4x11] : [true if file exists]
     * }</pre>
```

So yepp, the `@code` is what makes the difference here. It basically tells the paragraph of javadoc that the code inside this block is source code and should not be considered as Html tags

In case my explanation isn't good, maybe related  :  https://stackoverflow.com/questions/19302183/what-does-the-code-java-annotation-do#:~:text=%7B%40code%20...%7D%20is,differently%20than%20the%20other%20text.

Reply:
Thanks

Reply:
Done